pkg/status/obsolete_file_error.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package status
+
+// PathErrorCode is the error code for a generic PathError.
+const ObsoleteFileErrorCode = "2016"
+
+var ObsoleteFileErrorBuilder = NewErrorBuilder(ObsoleteFileErrorCode)
+
+// ObsoleteFileError returns a ObsoleteFileError wrapping an error one or more relative paths.
+func ObsoleteFileError(message string) Error {
```

Comment:
I think something like `NewCommitDetectedError` or similar might be a bit more precise

Reply:
CommitChangedError?

Reply:
Removed, using internal error instead